concept/impl/ConceptImpl.java

```diff
@@ -39,21 +41,21 @@
  */
 public abstract class ConceptImpl implements Concept, ConceptVertex {
     private final VertexElement vertexElement;
-    final ConceptManagerImpl conceptManager;
-    final ConceptObserver conceptObserver;
+    final ConceptManager conceptManager;
+    final ConceptNotificationChannel conceptNotificationChannel;
 
     //WARNING: DO not flush the current shard into the central cache. It is not safe to do so in a concurrent environment
     private final ConceptCache<Shard> currentShard;
```

Comment:
I know it's not related to your PR, but please delete the above WARNING comment if possible. We already decided to delete it, but we keep forgetting it. It's just outdated now. Fearmongering!